### DISCUSSION TITLE
"Australian" language option

### DISCUSSION THREAD

**Discussion by MajorFordson** [voxeltycoon/issues#461] [UI&UX]
If there are other volunteers (including myself) an option for an "Australian" translation would be good fun. It wouldn't replace every English word or phrase, but there's plenty of Australian slang or terminology that could be used.

**Comment by victorleblais** [comment]
If you want to volunteer, head over to the Discord server (https://discord.gg/voxeltycoon) and go to the #translation channel.

Once there, check the pinned messages and/or ping @refreshfr
